tooling/template/tsconfig.json

```diff
@@ -1,39 +1,20 @@
 {
+  "extends": "@isomer/tsconfig/base.json",
   "compilerOptions": {
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": true,
-    "noEmit": true,
-    "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "bundler",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
+    "lib": ["es2022", "dom", "dom.iterable"],
```

Comment:
i think teh change from `esnext` to `es2022` should be fine but is there anything that eg: `Next` relies on in `esnext`? or can just resolve this if you've used `es2022` with `Next` in previous projects

Reply:
typecheck should warn